.github/actions/python-environment/ext/get_poetry.py

```diff
@@ -0,0 +1,959 @@
+#!/usr/bin/env python3
+r"""
```

Comment:
a potential follow-up task would be how we update this, like in the developer guide.